packages/flutter_tools/lib/src/xcode_project.dart

```diff
@@ -498,7 +515,7 @@ class IosProject extends XcodeBasedProject {
         ? _flutterLibRoot
             .childDirectory('Flutter')
             .childDirectory('FlutterPluginRegistrant')
-        : hostAppRoot.childDirectory(XcodeBasedProject._hostAppProjectName);
+        : hostAppRoot.childDirectory(hostAppProjectName);
```

Comment:
This doesn't seem right either, before I renamed the Xcode project these files were at `ios/Runner/GeneratedPluginRegistrant.h` but on this PR after the rename it started generating it in `ios/Runner1` which isn't used anywhere.